examples/react/react-native-app/package.json

```diff
@@ -29,5 +28,6 @@
     },
     "devDependencies": {
         "@babel/plugin-proposal-decorators": "*"
-    }
+    },
+    "private": true
```

Comment:
I know this is just moved, but does it even need to be here?

Reply:
kinda? it basically tells the tooling that this can never be published

Reply:
It does - otherwise lerna tries to publish it to npm.